features/step_definitions/machine.rb

```diff
@@ -13,19 +13,30 @@
       logger.warn "We will skip this scenario"
       skip_this_scenario
     end
+
+  machine_sets = BushSlicer::MachineSetMachineOpenshiftIo.list(user: admin, project: project("openshift-machine-api"))
+    if machine_sets.length == 0
+      logger.warn "worker machineset does not exist, tests running by copying existing machinesets will fail."
```

Comment:
The logging is not precise, you are listing all machinesets from the `openshift-machine-api` but the logger says `worker machiensets'